### PR TITLE
Add Experimenters and Groups to JSON api docs

### PR DESCRIPTION
**Pull request by will-moore**
Document JSON api additions from https://github.com/ome/omero-web/pull/148

### CODE REVIEW

File: omero/developers/json-api.rst [hunk 1]
```diff
@@ -1278,6 +1278,169 @@ you can also show ROIs on an Image using this URL:
     GET     /api/v0/m/images/{image_id}/rois/
 
 
+Experimenters and ExperimenterGroups
+-------------------------------------
+
+Support for listing ROIs was added in API version 0.2.
```
Comment (sbesson) [inline omero/developers/json-api.rst]: Does the `Versioning` section of this page also need to be updated?

File: omero/developers/json-api.rst [hunk 2]
```diff
@@ -1197,8 +1197,8 @@ ROIs and Shapes
 
 Support for listing ROIs was added in API version 0.1.
 ROIs are linked to Images and contain one or more Shapes. Types of
-shape are Rectangle, Ellipse, Point, Line, Polyline, Polygon and
-Label. The Mask type is not currently supported by omero-marshal.
+shape are Rectangle, Ellipse, Point, Line, Polyline, Polygon, Label
```
Comment (jburel) [inline omero/developers/json-api.rst]: ordering the supporting shapes alphabetically will be easier to read

File: omero/developers/json-api.rst [hunk 3]
```diff
@@ -1278,6 +1278,169 @@ you can also show ROIs on an Image using this URL:
...
+-------------------------------------
+
+Support for listing ROIs was added in API version 0.2.
+Experimenters are users of OMERO and can belong to one or more ExperimenterGroups.
+
+Listing Experimenters
+^^^^^^^^^^^^^^^^^^^^^
+
+OMERO will only allow you to access details of Experimenters who are members of
+a non-private group that you are also a member of.
```
Comment (jburel) [inline omero/developers/json-api.rst]: emphasise ``non-private`` in bold

File: omero/developers/json-api.rst [hunk 4]
```diff
@@ -1278,6 +1278,169 @@ you can also show ROIs on an Image using this URL:
     GET     /api/v0/m/images/{image_id}/rois/
 
 
+Experimenters and ExperimenterGroups
```
Comment (jburel) [inline omero/developers/json-api.rst]: in the UI we talk about Experimenter and Groups
so maybe indicate that when retrieving via the API we have to use the "model" object ExperimenterGroup.

File: omero/developers/json-api.rst [hunk 5]
```diff
@@ -1278,6 +1278,169 @@ you can also show ROIs on an Image using this URL:
...
+
+::
+
+    GET   /api/v0/m/experimenters/{experimenter_id}/
+
+Experimenters in a Group
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Experimenters can be filtered by ExperimenterGroup using the ``?experimentergroup=id`` query string but
+you can also show Members of an ExperimenterGroup using this URL:
```
Comment (jburel) [inline omero/developers/json-api.rst]: show Members in a Group using...

File: omero/developers/json-api.rst [hunk 6]
```diff
@@ -1278,6 +1278,169 @@ you can also show ROIs on an Image using this URL:
...
+            }
+          },
+          "Name": "Swedlow Lab",
+          "url:experimentergroup": "http://server.openmicroscopy.org/web/api/v0/m/experimentergroups/10/",
+          "url:experimenters": "http://server.openmicroscopy.org/web/api/v0/m/experimentergroups/10/experimenters/"
+        },
+      ]
+    }
+
+Get a single ExperimenterGroup
```
Comment (jburel) [inline omero/developers/json-api.rst]: Group

**Review by jburel — COMMENTED** [review]
Need clarification about the mapping between Groups and model object ExperimenterGroup.

File: omero/developers/json-api.rst [hunk 7]
```diff
@@ -1278,17 +1278,18 @@ you can also show ROIs on an Image using this URL:
     GET     /api/v0/m/images/{image_id}/rois/
 
 
-Experimenters and ExperimenterGroups
+Experimenters and Groups
```
Comment (jburel) [inline omero/developers/json-api.rst:1281]: length of line needs to be adjusted accordingly

File: omero/developers/json-api.rst [hunk 8]
```diff
@@ -46,7 +46,7 @@ version number can be found in the header:
 
 ::
 
-    X-OMERO-ApiVersion: 0.1
+    Current Version : 0.2
```
Comment (sbesson) [inline omero/developers/json-api.rst]: Has the HTTP header key been modified?

Comment (will-moore) [reply omero/developers/json-api.rst]: Oooops - sorry, I was misreading and missed the "header" bit. Couldn't remember what the `X-OMERO-ApiVersion` referred to. Fixed now.

File: omero/developers/json-api.rst [hunk 9]
```diff
@@ -1278,6 +1278,170 @@ you can also show ROIs on an Image using this URL:
...
+              "isGroupRead": false,
+              "isGroupAnnotate": false,
+              "isUserWrite": false,
+              "isUserRead": false
+            }
+          },
+          "FirstName": "Ben",
+          "LastName": "Nevis",
+          "UserName": "ben",
+          "url:experimenter": "http://server.openmicroscopy.org/web/api/v0/m/experimenters/10/",
```
Comment (jburel) [inline omero/developers/json-api.rst]: better to have a url with ``https``

File: omero/developers/json-api.rst [hunk 10]
```diff
@@ -1278,6 +1278,170 @@ you can also show ROIs on an Image using this URL:
...
+          "url:experimentergroups": "http://server.openmicroscopy.org/web/api/v0/m/experimenters/10/experimentergroups/"
```
Comment (jburel) [inline omero/developers/json-api.rst]: same

File: omero/developers/json-api.rst [hunk 11]
```diff
@@ -1278,6 +1278,170 @@ you can also show ROIs on an Image using this URL:
...
+              "isGroupWrite": false,
+              "isGroupRead": false,
+              "isGroupAnnotate": false,
+              "isUserWrite": false,
+              "isUserRead": false
+            }
+          },
+          "Name": "Swedlow Lab",
+          "url:experimentergroup": "http://server.openmicroscopy.org/web/api/v0/m/experimentergroups/10/",
+          "url:experimenters": "http://server.openmicroscopy.org/web/api/v0/m/experimentergroups/10/experimenters/"
```
Comment (jburel) [inline omero/developers/json-api.rst]: same

**Comment by jburel** [comment]
Looks good thanks @will-moore 
This should be merged when the corresponding code PR is merged

**Comment by jburel** [comment]
Travis is green see https://travis-ci.org/github/ome/ome-documentation/builds/706146567
but the synch did not happen.
Code PR has been merged, merging the doc

**Pull request merged**